### ISSUE TITLE
[BUG]: Dashboard filters use "FieldName" rather than "FieldLabel"

### ISSUE THREAD

**Issue by exscalliber** [RevealBi/Reveal.Sdk#203]
### SDK Version

1.6.6

### Client Framework

HTML/JavaScript

### Server Platform

Node.js

### Operating System

Mac

### Description

Hi, I have been playing with the "onFieldsInitializing" event on the revealView on the client SDK and have found that it does not affect the filters. A good example is when you add a dashboard scoped "Dashboard Filter". If you change the labels of the fields, it will not be reflected in the filter and will instead show the "FieldName" instead of the "FieldLabel" (im referencing the getSchema endpoint return values). 

To change the field names im doing this in the revealView:

```
revealView.onFieldsInitializing = function (args) {
  args.fields.forEach(field => {
    if (field.name === 'ABC') {
      field.label = 'Pretty Name'
    } else if (...
  });
}
```

This changes the field names in the edit visualization screen perfectly but does not change filters.

**Comment by luispandolfi** [comment]
Thanks @exscalliber for reporting.
We already created an internal issue to track it AB#30885
We'll keep you updated here.

**Comment by brianlagunas** [comment]
This has been fixed and will be available in our 1.7.0 release.
